src/python/pants/rules/core/BUILD

```diff
@@ -1 +1,17 @@
 python_library()
+
+python_tests(
+  name = 'roots',
+  sources = ['test_list_roots.py'],
+  dependencies = [
+    'src/python/pants/backend/graph_info/tasks',
```

Comment:
Where did these deps come from? You only need to depend on the targets containing the things your test file imports.